pkgs/development/python-modules/mocket/default.nix

```diff
@@ -1,10 +1,9 @@
-{ lib, buildPythonPackage, fetchPypi, pythonOlder, isPy27
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, isPy3k
 , decorator
 , http-parser
 , importlib-metadata
```

Comment:
```suggestion
```

Reply:
It took me a while to figure out that `nixpkgs-review-checks` is detecting that.